xml/Microsoft.CLRAdmin/CDO.xml

```diff
@@ -305,12 +378,26 @@
         <Parameter Name="c" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="a">To be added.</param>
-        <param name="b">To be added.</param>
-        <param name="c">To be added.</param>
+        <param name="a">A pointer to the <see cref="T:Microsoft.CLRAdmin.FORMATETC" /> defining the format for the data object implementing the <see cref="T:Microsoft.CLRAdmin.IDataObject" /> to use when interpreting the data contained in the storage medium.</param>
+        <param name="b">A pointer to the <see cref="T:Microsoft.CLRAdmin.STGMEDIUM" /> defining the storage medium for the SetData method to use to pass the data.</param>
+        <param name="c">
+          <see langword="true" /> to specify that the called data object implementing the <see cref="T:Microsoft.CLRAdmin.IDataObject" /> owns the storage medium after the call and must free the medium after it has been used; otherwise, <see langword="false" /> to specify the caller retains ownership of the storage medium and the data object uses the medium for the duration of the call only.</param>
```

Comment:
```suggestion
          <see langword="true" /> to specify that the called <see cref="T:Microsoft.CLRAdmin.CDO" /> object owns the storage medium after the call and must free the medium after it has been used; otherwise, <see langword="false" /> to specify the caller retains ownership of the storage medium and the data object uses the medium for the duration of the call only.</param>
```